lib/Echidna/ABI.hs

```diff
@@ -344,3 +346,8 @@ genAbiCallM genDict abi = do
 -- | Given a list of 'SolSignature's, generate a random 'SolCall' for one, possibly with a dictionary.
 genInteractionsM :: MonadRandom m => GenDict -> NE.NonEmpty SolSignature -> m SolCall
 genInteractionsM genDict l = genAbiCallM genDict =<< rElem l
+
+abiCalldata :: Text -> Vector AbiValue -> BS.ByteString
+abiCalldata s xs = BSLazy.toStrict . runPut $ do
```

Comment:
Is there any way to avoid calling `toStrict` ?

Reply:
this function is copied from `hevm` as it was missing in the newer version, why it is bad? what would you suggest instead?